net.sf.eclipsecs.core/src/net/sf/eclipsecs/core/config/meta/MetadataFactory.java

```diff
@@ -346,6 +592,71 @@ private static void doInitialization() throws CheckstylePluginException {
         CheckstyleLog.log(e, "Could not read metadata " + metadataFile); //$NON-NLS-1$
       }
     }
+
+    fillWithRemainingMetadata();
+  }
+
+  /**
+   * Create repository of Module Details from checkstyle metadata and sevntu checks metadata.
+   */
+  public static void createMetadataMap() {
+    new XMLMetaReader().readAllModulesIncludingThirdPartyIfAny(sPackageNameSet
+          .toArray(new String[sPackageNameSet.size()]))
+        .forEach(moduleDetail -> sModuleDetailsRepo.put(moduleDetail.getName(), moduleDetail));
+  }
+
+  /**
+   * Creates new RuleMetadata which are absent in the eclipse-cs metadata package.
+   */
+  public static void fillWithRemainingMetadata() {
```

Comment:
please always load all metadata and override data if it was loaded from old files.
metadata from checkstyle has MAX priority of usage.

please rename method to be `loadRuleMetadatas`

Reply:
Yes, its happening in that fashion, I have updated the name